server/controller/apis/topicController.js

```diff
@@ -278,33 +247,83 @@ exports.saveTopic = async ( req, res, redirect ) => {
 
         if(topic.topicType == 3) {
             
-            topic.topicContentHtml = req.body.embedded_submission_text_topic;
+            topic.topicHtml = req.body.embedded_submission_text_topic;
         }
         else {
             // check to see if the incomping message format is from the UI or the API
-            if( req.body.topicContentHtml ) {
-                topic.topicContentHtml = req.body.topicContentHtml;
+            if( req.body.topicHtml ) {
+                topic.topicHtml = req.body.topicHtml;
             }
             else {
-                topic.topicContentHtml = req.body.topicEditor;
+                topic.topicHtml = req.body.topicEditor;
```

Comment:
This was a really good catch; I was worried that the .ejs files would be sending a field named topicContentHtml, but a review of those files looks like it sends topicHtml as adjusted here. Well done.